src/multiform/MultiForm.ts

```diff
@@ -134,13 +135,17 @@ export default class MultiForm extends AVisInstance implements IVisInstance, IMu
         };
         this.actVis.on('transform', t);
         const r = this.actVis.transform(scale, rotate);
+        this.options.all.heightTo = this.actVis.rawSize[1] * (scale[1]||1);
```

Comment:
With the edited visualizations it should work everywhere, it is not dependent on mother table code in any way. Though you are right  that `this.actVis.size[1]` should be enough.